elasticgraph-graphql/spec/unit/elastic_graph/graphql/query_adapter/requested_fields_spec.rb

```diff
@@ -98,7 +98,9 @@ class QueryAdapter
               t.relates_to_many "parts", "Part", via: "part_ids", dir: :out, singular: "part"
               t.field "owner_id", "ID", indexing_only: true
               t.field "owner", "ComponentOwner", name_in_index: "owner_id", graphql_only: true
-              t.index "components"
+              t.index "components" do |i|
+                i.has_had_multiple_sources!
```

Comment:
Same here--doesn't seem like this is needed. Before merging it'd be nice to reduce the `has_had_multiple_sources!` calls in tests to just the ones where it's needed.